json-parser/src/main/java/org/uniprot/core/json/parser/uniparc/UniParcCrossRefJsonConfig.java

```diff
@@ -48,7 +49,9 @@ public ObjectMapper getFullObjectMapper() {
 
     @Override
     public ObjectMapper getSimpleObjectMapper() {
-        return this.prettyMapper;
+        ObjectMapper simpleObjectMapper = getDefaultSimpleObjectMapper();
+        simpleObjectMapper.registerModule(new JavaTimeModule());
```

Comment:
Can you please move these lines to initPrettyObjectMapper() method?
and in this method you can keep return this.prettyMapper;